Wikia/selenium-tests
# Concf 801

Acceptance Criteria:
* check if tap on elements from different namespaces redirects to correct pages
** check if article loads inside mercury
** check if other items redirect outside mercury (except category) - check class on body, skin JS variable or more ways to check it
* test are executed daily

Algorithm:
* open http://mercurycc.wikia.com/main/Category:Articles
* check icon on image is #namespace-article
* tap on first element
* check url pattern /wiki/

* open http://mercurycc.wikia.com/main/category/Blog_posts
* check icon on image is #namespace-blog
* tap on first element
* check url pattern /wiki/User_blog:

* open http://mercurycc.wikia.com/main/category/Images
* check icon on image is #namespace-image
* tap on first element
* check url pattern /wiki/File:

* open http://mercurycc.wikia.com/main/category/Videos
* check icon on image is #namespace-video
* tap on first element
* check url pattern /wiki/File:
